jobs/settings/development.py

```diff
@@ -1,5 +1,16 @@
 from . import *  # noqa
 
+DATABASES = {
+    'default': {
```

Comment:
Black is complaining because have to be double quote. Run black and it will change everything as it wants.